index.html

```diff
@@ -11,6 +11,12 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
```

Comment:
Hey little baby, please fix these google font imports!

Reply:
Thanks for removing them!

Reply:
I did it!